content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -54,6 +54,13 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    *Note* if you are running `kubectl` on Windows, you'd need to use double quotes on the
+    command line and thus escape the inner double quotes like this:
```

Comment:
Would you mind altering that text to use a `{{< note >}}` [shortcode](https://kubernetes.io/docs/contribute/style/style-guide/#shortcodes)? That fits in better with the website style guidelines.

Reply:
Thanks @sftim , I've placed the note inside a shortcode.